packages/core/src/sharedWishlists/redux/__tests__/reducer.test.js

```diff
@@ -270,6 +270,15 @@ describe('shared wishlists reducer', () => {
       ).toBe(expectedResult);
     });
 
+    it.only('should handle REMOVE_SHARED_WISHLIST_SUCCESS action type', () => {
+      expect(
+        reducer(currentState, {
+          payload: { result: 'foo' },
```

Comment:
Also, the action will not contain a `payload` property when dispatched by the `doDeleteSharedWishlist.js` action.

Reply:
Right! Updated